src/ApiPort/CommandLineOptions.cs

```diff
@@ -47,11 +48,11 @@ private static ICommandLineOptions ShowHelp(string suppliedCommand = null)
                 Console.WriteLine();
             }
 
-            Console.WriteLine("Available Commands:");
-
             foreach (var command in s_possibleCommands)
             {
-                Console.WriteLine($"- {command.Key}");
+                Console.WriteLine();
+                Console.WriteLine(new string('=', Math.Min(Console.WindowWidth, 100)));
```

Comment:
If you have WindowWidth characters, won't the \r\n from WriteLine wrap to the next line so that if WindowWidth is >100, we end up with the next line directly below this one and with WindowWidth <= 100, we end up with a blank line in between?

Like I said, this is very nitpicky :)